dev-client/src/components/home/BottomSheet.tsx

```diff
@@ -15,9 +15,7 @@ type SiteListSiteProps = {
 const SiteListSite = ({site, showSiteOnMap}: SiteListSiteProps) => {
   const {t} = useTranslation();
   const project = useSelector(state =>
-    site.projectId === undefined
-      ? undefined
-      : state.project.sites[site.projectId],
+    site.projectId === undefined ? undefined : state.site.sites[site.projectId],
```

Comment:
nit:
```suggestion
    site.projectId === undefined ? undefined : state.project.projects[site.projectId],
```
this was my bad, i fixed a silly mistake in https://github.com/techmatters/terraso-client-shared/pull/29 and didn't change the downstream weirdness created by that mistake. i think  i also fixed this in the screen refactor i just merged so it should be good if you rebase